src/Term/TermList.php

```diff
@@ -116,7 +116,12 @@ private function assertIsLanguageCode( $languageCode ) {
 	}
 
 	public function setTerm( Term $term ) {
```

Comment:
Please describe the new behaviour in the documentation of the constructor and this method.